app/sdks/paypal_client.rb

```diff
@@ -0,0 +1,70 @@
+require 'paypal-checkout-sdk'
+
+class PaypalClient
+  def self.create_payment(orders:)
```

Comment:
the `orders` params should have default value in case there's no input args and lead to failing `.each` below
```
def self.create_payment(orders: [])
```

Reply:
noted and updated